src/app/models/user-details.model.ts

```diff
@@ -8,13 +8,20 @@ export interface UserInfo {
   uid?: string;
 }
 
+export interface LocationInfo {
+  primaryLocation: string;
+  jurisdiction: string;
+  isCaseAllocator: boolean;
+}
+
 export interface UserDetails {
   sessionTimeout: {
     idleModalDisplayTime: number,
     totalIdleTime: number,
   };
   canShareCases: boolean;
   userInfo: UserInfo;
+  locationInfo: LocationInfo []
```

Comment:
would this be optional? Solicitors have this locationInfo?
This UserDetails is used by all user type not only for caseworker/judicial users.

Reply:
@rayliangatos  it will be a blank array for them. I have initialised it already. Also, tested it

Reply:
Done